tests/__tests__/refmterr/refmterr_test.re

```diff
@@ -170,49 +170,20 @@ let forEachTest =
         ),
       ),
     );
-    /* open the produced error output */
-    let expected = readFile(expectedOutputName);
-    let actual = readFile(actualOutputName);
+
     /* swap-comment below two lines if you want to generate new expected
        from the new actual */
     /* ignore @@ Sys.command @@ Printf.sprintf("cp %s %s", actualOutputName, expectedOutputName); */
-    /* TODO: show the differences */
-    if (actual != expected) {
-      print_endline("The test output does not match what was expected.");
-      print_newline();
-      print_endline(
-        "----------------------------------------------------------------------",
-      );
-      print_endline("Actual:" ++ actualOutputName);
-      print_endline(
-        "----------------------------------------------------------------------",
-      );
-      print_endline(actual);
-      print_endline(
-        "----------------------------------------------------------------------",
-      );
-      print_newline();
-      print_newline();
-      print_endline(
-        "----------------------------------------------------------------------",
-      );
-      print_endline("Expected:" ++ expectedOutputName);
-      print_endline(
-        "----------------------------------------------------------------------",
-      );
-      print_endline(expected);
-      print_endline(
-        "----------------------------------------------------------------------",
-      );
-      print_newline();
-      print_newline();
-      changeCommands.contents = [
-        "cp " ++ actualOutputName ++ " " ++ expectedOutputName,
-        ...changeCommands.contents,
-      ];
-      print_newline();
-      /* raise(Not_equal(filename)); */
-    };
+
+    test(
+      dirname,
+      ({expect}) => {
+        /* open the produced error output */
+        let expected = readFile(expectedOutputName);
+        let actual = readFile(actualOutputName);
+        expect.string(actual).toEqual(expected);
```

Comment:
This could be refactored to expect.file(actualOutputName).toMatchSnapshot(). This would cause the expected output to be managed by Rely snapshots instead of this directory, which would make them easier to update.

Reply:
I would love to do snapshot testing with `expect.file` and remove all the fixtures. But, since we need to hacky-bypass the readFile for an explicit case: `file_SyntaxError/file_SyntaxError_7.re`. 

I'm afraid we can't do it yet.